website/docs/manage_groups.md

```diff
@@ -51,4 +60,59 @@ More levels of folder nesting are not supported.
 │   └─Japanese   <- Group
 |       └─DictB  <- Dict Files's container folder
 |          └─ DictB Files  
-```
\ No newline at end of file
+```
+
+## group by metadata.txt
+
+Click the "group by metadata" will group your dicts based on metadata.txt file.
+
+the location of the metadata file is at the same location as the dictionary.
+
+The structure of the metadata file use [toml](https://toml.io/en/) format
+```
+category = [ "english", "yellow", "green" ]
+
+# the following fields have not supported yet.
+[metadata]
+name = "New Name"
+langfrom = "English"
+langto = "Russian"
+```
+
+
+For example, 
+
+```
+.
+English<>Chinese
+── Cambridge
+   ├── Cambridge.dict.dz
+   ├── Cambridge.idx
+   ├── Cambridge.info
+   └── Cambridge.syn
+   └── metadata.txt     (A)
+── Collins
+   ├── Collins.dsl
+   └── res.zip
+   └── metadata.txt (B)  
+
+```
+
+The content of the metadata(A) is
+```
+category=["en-zh","汉英词典"]
+```
+
+
+The content of the metadata(B) is
+```
+category=["图片词典","en-zh","汉英词典"]
+```
+
+the structure below will be auto grouped into three groups:
+
+* `en-zh` with `Cambridge`, `Collins` 
+* `图片词典` with `Collins` 
+* `汉英词典` with `Cambridge`,`Collins` 
+
+Note: you can configure only the dictionaries you like .
```

Comment:
Sorry, but what do you mean here?

Maybe "Dictionaries without `metadata.toml` won't be auto-grouped."?

Reply:
yes,  If users has not set metadata.toml ,the dictionary should be ignored.

Reply:
I think "Dictionaries without `metadata.toml` won't be auto-grouped." is better because "dictionaries you like" is too subjective :sweat_smile: